packages/search/README.md

```diff
@@ -14,7 +14,10 @@
 
 # @esri/hub-search
 
-> Module to search for ArcGIS items and format them for display in ArcGIS Hub.
+> **DEPRECATED**: This package is now deprecated and currently only includes functions that are still in use in legacy applications.
+Use the `hubSearch()` function and related classes in `@esri/hub-common` instead.
```

Comment:
this not able to search the hub index at this point... I think that's ok, but maybe we should note that it's not a 100% replacement at this point